gulpfile.js

```diff
@@ -12,12 +20,83 @@ export const styles = () => {
     .pipe(plumber())
     .pipe(less())
     .pipe(postcss([
-      autoprefixer()
+      autoprefixer(),
+      csso()
     ]))
+    .pipe(rename('style.min.css'))
     .pipe(gulp.dest('source/css', { sourcemaps: '.' }))
```

Comment:
Каталог назначения должен быть `build`. Проект теперь собирается там.
```suggestion
    .pipe(gulp.dest('build/css', { sourcemaps: '.' }))
```